velocity.js

```diff
@@ -65,6 +66,13 @@ const loginPage = 'login';
 const modeSelectorPage = 'modeSelector';
 const pageNotFoundPage = 'pageNotFound';
 const profilePage = 'profile';
+const usersPage = 'users/users';
+const usersAddPage = 'users/users-add';
+const usersEditPage = 'users/users-edit';
+const usersImportCompletePage = 'users/users-import-complete';
+const usersImportPage = 'users/users-import';
```

Comment:
Should we make it /users/ instead of users/ given earlier changes as such?

Reply:
there is a difference. /users is to redirect the path to the absolute path in the UI which is required to avoid hitting a random 404 page. But in the backend, we need the relative path with in the Templates folder so it can be used by the templating engine